test/unittests/utils/test_filemanip.py

```diff
@@ -6,6 +6,62 @@
 from numpy.testing import assert_array_equal
 
 
+@pytest.mark.parametrize(
+    "dictionary, expected",
+    [
+        (
+            {
+                "TotalReadoutTime": 1,
+                "EstimatedTotalReadoutTime": "",
+                "PhaseEncodingSteps": "",
+                "PixelBandwidth": "",
+                "PhaseEncodingDirection": "j+",
+                "PhaseEncodingAxis": "",
+            },
+            [1, "j+"],
+        ),
+        (
+            {
+                "EstimatedTotalReadoutTime": 1,
+                "PhaseEncodingDirection": "j",
+            },
+            [1, "j"],
+        ),
+        (
+            {
+                "PhaseEncodingSteps": 1,
+                "PixelBandwidth": 0.5,
+                "PhaseEncodingDirection": "j-",
+            },
+            [2, "j-"],
+        ),
+    ],
+)
+def test_extract_metadata_from_json(dictionary, expected):
+    """This function tests something."""
```

Comment:
That is true but I'd try to be a bit more specific... ;)

Reply:
whouups